### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by sansive**
## Task
* Show no info message when there is no household selected
* Add create household test

### CODE REVIEW

**Review by Geno22 — APPROVED** [review]
Nice thanks for adding this

File: .idea/misc.xml [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="..\:/Users/sansi/AndroidStudioProjects/House-Organizer/app/src/main/res/layout/activity_info.xml" value="0.13858695652173914" />
+        <entry key="..\:/Users/sansi/AndroidStudioProjects/House-Organizer/app/src/main/res/layout/activity_main_screen.xml" value="0.13858695652173914" />
```
Comment (Defibrilator) [inline .idea/misc.xml:7]: please check the paths before pushing misc.xml

**Review by Defibrilator — CHANGES_REQUESTED** [review]
Good addition, thank you

**Review by Oxydeas — APPROVED** [review]

**Review by Oxydeas — COMMENTED** [review]
@Defibrilator Your change request is blocking this PR

**Review by Defibrilator — APPROVED** [review]
Thank you for the changes

**Pull request merged**